examples/asr/transcribe_speech.py

```diff
@@ -251,7 +277,9 @@ def autocast():
             with open(cfg.dataset_manifest, 'r') as fr:
                 for idx, line in enumerate(fr):
                     item = json.loads(line)
-                    item['pred_text'] = transcriptions[idx]
+                    item['pred_text'] = transcriptions[idx].text
```

Comment:
Isn't this only if hypothesis objects is returned ?

Reply:
sorry. fixed.